src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Notification.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { type FC } from 'react';
+
+import { useGetFullColonyByAddressQuery } from '~gql';
+import {
+  NotificationType,
+  type Notification as NotificationInterface,
+} from '~types/notifications.ts';
+
+import ActionNotification from './Action/ActionNotification.tsx';
+import ExpenditureNotification from './Expenditure/ExpenditureNotification.tsx';
+
+const displayName = 'common.Extensions.UserHub.partials.Notification';
+
+interface NotificationProps {
+  notification: NotificationInterface;
+}
+
+const Notification: FC<NotificationProps> = ({ notification }) => {
+  const { data: colonyData, loading: loadingColony } =
+    useGetFullColonyByAddressQuery({
```

Comment:
Could this use a separate leaner query? At glance only the native token and metadata are necessary

Reply:
Great idea, done!